index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>mobile-project</title>
+</head>
+<body>
+  <header>
+  <nav class="flex-nav">
+    <div>
+    <img src="images/MyLogo.png" alt="logo">
+    </div>
+    <div>
+    <img src="images/Portoflio.png" alt="Portoflio" class="right">
+    <img src="images/About.png" alt="About" class="right">
+    <img src="images/Contact.png" alt="Contact" class="right">
+  </div>
```

Comment:
- [ ] The logo on the left should have a link `a` tag. This aids the webpage navigation and improves the overall aesthetics and user experience.
- [ ] You are missing the hamburger button on the right. See the images below for reference. 
_You must stick to the design as much as possible (e.g. font, colours, images, margins) using the templates in Figma._

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/120888366-cda32500-c5ac-11eb-9c52-f765de221961.png) | ![image](https://user-images.githubusercontent.com/49621279/120888386-eb708a00-c5ac-11eb-8aa2-89ef24e36fb7.png)

Reply:
These issues raised by the previous TSE still persist. Please make sure you made all the requested changes